jax/core.py

```diff
@@ -1025,8 +1025,6 @@ def concrete_or_error(force: Any, val: Any, context=""):
   else:
     return force(val)
 
-convert_element_type_p = Primitive('convert_element_type')
```

Comment:
This was only moved here in #6014 out of necessity, because we had special-cased some jaxpr simplifications on it in partial_eval.py.